panther_analysis_tool/main.py

```diff
@@ -535,12 +598,15 @@ def filter_analysis(analysis: List[Any], filters: Dict[str, List]) -> List[Any]:
 
 def classify_analysis(
     specs: List[Tuple[str, str, Any, Any]]
-) -> Tuple[List[Any], List[Any], List[Any], List[Any]]:
+) -> Tuple[Dict[str, List[Any]], List[Any]]:
+
+    # First setup return dict containing different
+    # types of detections, meta types that can be zipped
+    # or uploaded
+    classified_specs = dict()
+    for key in [DATAMODEL, DETECTION, GLOBAL, PACK]:
+        classified_specs[key] = []
```

Comment:
The list of returned things was getting quite long - so I made it into a dict. Rather than adding yet another object to return it will simply return and dict. If there are more things to add in the future, we can simply add it as a new key in the dict.  Wondering if that seems logical?